docker.py

```diff
@@ -60,6 +83,11 @@ def args_create(argv):
         dargs.append('--label=xrootd-local-gateway=true')
         dargs.append('--network=ralworker')
         dargs.append('--add-host=xrootd.echo.stfc.ac.uk ceph-gw10.gridpp.rl.ac.uk ceph-gw11.gridpp.rl.ac.uk:172.28.1.1')
+        # Call function to capture primary IPv6 address and assign xrootd alias to local containers IPv6 address.
+        primary_ipv6 = get_primary_ipv6()
+        if primary_ipv6:
+            primary_ipv6 = primary_ipv6.rstrip(':')
+            dargs.append('--add-host=xrootd.echo.stfc.ac.uk ceph-gw10.gridpp.rl.ac.uk ceph-gw11.gridpp.rl.ac.uk:{}{}'.format(primary_ipv6, ':1000:2'))
```

Comment:
could you add svc20 too in this list? it got added as a aaa host recently

Reply:
tbd in a separate commit

Reply:
We can include this as a separate commit, this should just be a mirror of what we currently do for IPv4, we can diverge at a later time.